robot_ws/src/cloudwatch_robot/package.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version='1.0' encoding='utf-8'?>
 <package format="3">
   <name>cloudwatch_robot</name>
-  <version>2.0.0</version>
+  <version>2.0.1</version>
```

Comment:
So I take it based on https://github.com/aws-robotics/aws-robomaker-sample-application-cloudwatch/pull/69#discussion_r410378567, this should also be `2.1.0`?

Reply:
Yup!